libsql-server/src/query_analysis.rs

```diff
@@ -108,6 +109,7 @@ impl StmtKind {
             }) => Some(Self::Write),
             Cmd::Stmt(Stmt::DropView { .. }) => Some(Self::Write),
             Cmd::Stmt(Stmt::Savepoint(_)) => Some(Self::Savepoint),
+            Cmd::Stmt(Stmt::Vacuum(_, _)) => Some(Self::Vacuum),
```

Comment:
there's a mirror parser in our client code - `libsql/src/parser.rs`, and it should get updated in there as well